openai/openai_test.go

```diff
@@ -41,25 +35,28 @@ func captureRequestMiddleware(capturedRequest any) gin.HandlerFunc {
 func TestChatMiddleware(t *testing.T) {
 	type testCase struct {
 		Name     string
-		Setup    func(t *testing.T, req *http.Request)
-		Expected func(t *testing.T, req *api.ChatRequest, resp *httptest.ResponseRecorder)
+		Input    string
```

Comment:
```suggestion
		body string
```

Might be clearer - the request body